staging/src/k8s.io/kubectl/pkg/cmd/get/get.go

```diff
@@ -513,14 +511,9 @@ func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 		objs[ix] = infos[ix].Object
 	}
 
-	sorting, err := cmd.Flags().GetString("sort-by")
```

Comment:
Now we are removing this line because it is already set in `Complete`. But we are not executing `Complete` function in here https://github.com/kubernetes/kubernetes/blob/774ab1349bfd553c97a99b992e9f38659f541c84/staging/src/k8s.io/kubectl/pkg/util/completion/completion.go#L180

Maybe completions do not use sort-by flag and this change will not create any problem?

Reply:
I don't think allowing users to customize sorting for completions is really necessary. This seems fine to me.